### ISSUE TITLE
Alert is not send when specific price is changed

### ISSUE THREAD

**Issue by getdatakick** [getdatakick/pricealert#2]
If product have specific prices, email alert is not send when this changes.

**Comment by kokoonshop** [comment]
Will be great to add this function, thx

**Comment by backamblock** [comment]
has anything changed about the integration of this feature? i work with specific prices and discounts a lot but not changing the real product price :(

**Comment by getdatakick** [comment]
Not really, I don't have time to work on pricealert module at the moment, so this task is on a back burner

**Comment by backamblock** [comment]
alright, no problem, not that important right now :)

**Comment by therampagerado** [comment]
For me this also is vital of using this module. Otherwise it's pretty much useless if it does not account for specific prices.

We want to show that those products are 'on sale' so to be more appealing to our customers. I doubt anybody changes the real price for discounts.

Have to disable it until this is fixed because it's not working at the moment.

I'll follow on it! Thank you for your great modules and TB work!

**Comment by mrabsinthe** [comment]
+1 we are also not changing real prices but applying specific prices quite often. Please consider this as it is really great module!
